From 20880ed3c176b400df548c84db13355c7c7007e4 Mon Sep 17 00:00:00 2001
From: Niko Tyni <ntyni@debian.org>
Date: Thu, 14 Aug 2014 22:02:18 +0000
Subject: [PATCH 2/3] Temporarily skip tests in t/01_load.t when 'use Wx
 qw(:allclasses)' fails

This is a workaround for wxwidgets3.0 breakage, see
https://bugs.debian.org/758127
---
 t/01_load.t | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/t/01_load.t b/t/01_load.t
index 2237a97..c4042c9 100755
--- a/t/01_load.t
+++ b/t/01_load.t
@@ -11,7 +11,8 @@ my $x = wxYES;
 ok( 1, "Exported constant" );
 
 SKIP: {
-  use Wx qw(:frame :allclasses wxNO_3D wxTAB_TRAVERSAL);
+  eval "use Wx qw(:frame :allclasses wxNO_3D wxTAB_TRAVERSAL)";
+  skip("loading :allclasses et al. failed: $@", 2) if $@;
 
   $x = wxTAB_TRAVERSAL();
   $x = wxCAPTION();
-- 
2.1.0.rc1

